### DISCUSSION TITLE
Missing Armor Model File

### DISCUSSION THREAD

**Discussion by Noggog** [Mutagen-Modding/Mutagen.Bethesda.Analyzers#82] [Topic Codes]
# Context
`Armor.WorldModel.[M/F].Model.File`
`ARMO/[MOD2/MOD4]`

# Description
Has a model listed, but the asset cannot be found in the load order

# Effects
Dropped armor is invisible

**Comment by Elscrux** [comment]
See this: https://ck.uesp.net/wiki/Armor
It is only an issue with both or only the female armor addon is assigned.
